test/unit/api/api-record-log-events.test.js

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const tap = require('tap')
+const helper = require('../../lib/agent_helper.js')
+const API = require('../../../api.js')
+const { SUPPORTABILITY, LOGGING } = require('../../../lib/metrics/names')
+const API_METRIC = SUPPORTABILITY.API + '/recordLogEvent'
+
+tap.test('Agent API - recordCustomEvent', (t) => {
+  t.autoend()
+
+  let agent = null
+  let api = null
+  const message = 'just logging a log in the logger'
+
+  t.beforeEach(() => {
+    agent = helper.loadMockedAgent()
+    api = new API(agent)
+  })
+
+  t.afterEach(() => {
+    helper.unloadAgent(agent)
+    agent = null
+    api = null
+  })
+
+  t.test('can handle a singular log message', (t) => {
+    const now = Date.now()
+    t.doesNotThrow(() => {
```

Comment:
this reads slightly odd, especially since there's nowhere in the API method that throws errors.

Reply:
Alright, removed this check and just called the function.